zerver/tests/test_zcommand.py

```diff
@@ -81,3 +84,75 @@ def test_fixed_zcommand(self) -> None:
         result = self.client_post("/json/zcommand", payload)
         self.assert_json_success(result)
         self.assert_in_response('You are still in fixed width mode', result)
+
+    def test_digress_zcommand(self) -> None:
+        self.login('hamlet')
+        hamlet = self.example_user('hamlet')
+        hamlet.save()
```

Comment:
I don't think we need hamlet.save here. We are not modifying any property of hamlet.

Reply:
Right, fixed.